### PR TITLE
Add tests for media replace flow

### LINKED ISSUE #19245: MediaReplaceFlow component has no tests
**Is your feature request related to a problem? Please describe.**
There are no tests in MediaReplaceFlow.

**Describe the solution you'd like**
There should be tests. It is a component that will affect a lot of blocks.

**Describe alternatives you've considered**
There are no alternatives.

### PR DESCRIPTION
**Pull request by draganescu**
Closes #19245

Adds the following unit tests for the media replace flow control:

- renders successfully
- renders replace options menu
- displays media URL
- edits media URL

### CODE REVIEW

File: packages/block-editor/src/components/media-replace-flow/test/index.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { act } from 'react-dom/test-utils';
```
Comment (Mamaduka) [inline packages/block-editor/src/components/media-replace-flow/test/index.js]: Is there any reason for not using `act` and `render` from the `@testing-library/react`?

Comment (draganescu) [reply packages/block-editor/src/components/media-replace-flow/test/index.js]: Me being a noob.

Comment (Mamaduka) [reply packages/block-editor/src/components/media-replace-flow/test/index.js]: I basically check this article every time I've to write unit tests 😅  - https://kentcdodds.com/blog/common-mistakes-with-react-testing-library

File: packages/block-editor/src/components/media-replace-flow/test/index.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
```
Comment (Mamaduka) [inline packages/block-editor/src/components/media-replace-flow/test/index.js]: We can also get rid of `noop` with something like - `const noop = () => {};`

**Review by Mamaduka — APPROVED** [review]
This looks good. Thanks, @draganescu.

Left another small comment, but feel free to ignore it :)

File: packages/block-editor/src/components/media-replace-flow/test/index.js (hunk 2 above)
Comment (draganescu) [reply packages/block-editor/src/components/media-replace-flow/test/index.js]: A good idea which I followed 👌🏻

**Pull request merged**